tests/casp/oci_role.pm

```diff
@@ -23,8 +23,17 @@ sub run() {
     send_key_until_needlematch "system-role-$role", 'down', 2;
     send_key 'ret' if (check_var('VIDEOMODE', 'text'));
 
-    # Set dashboard url for worker
-    if ($role eq 'worker') {
+    if ($role eq 'admin') {
+        # Try without ntp servers
+        send_key 'alt-i';
+        handle_simple_pw;
+        assert_screen 'ntp-servers-missing';
+        send_key 'alt-n';
+
+        send_key 'alt-t';
+        type_string 'ntp1 ntp2';
```

Comment:
I don't like hardcoded dependencies on external services - Would prefer a supportserver with ntp to avoid problems with tests failing if ntp1 ntp2 are down, or if these tests are running elsewhere where ntp1 and ntp2 to not work